### ISSUE TITLE
Change ModelCheckpoint.save_best_only without changing any other behaviour

### ISSUE THREAD

**Issue by fellnerse** [williamFalcon/pytorch-lightning#380]
How do I change the default behaviour `ModelCheckpoint.save_best_only=False` to True without changing any other behaviour, like where the checkpoints should be stored, or logging dir etc. When creating my own `ModelCheckpoint` like described in the docs, will delete the os.getcwd() directory and put the checkpoint there: 
```from pytorch_lightning.callbacks import ModelCheckpoint

# DEFAULTS used by the Trainer
checkpoint_callback = ModelCheckpoint(
    filepath=os.getcwd(),
    save_best_only=True,
    verbose=True,
    monitor='val_loss',
    mode='min',
    prefix=''
)

trainer = Trainer(checkpoint_callback=checkpoint_callback)
```
I have no Idea how to get the actual logging dir from the trainer, as I have to instantiate the Trainer with the callback. Couldn't find any easy nice solution without doing some nasty stuff. Can you help me here?
This seems not to work either..
``` 
     log_dir = "/log"
    checkpoint_callback = ModelCheckpoint(
        filepath=log_dir + "/checkpoints",
        save_best_only=True,
        verbose=True,
        monitor="val_acc",
        mode="min",
        prefix="",
    )

    trainer = Trainer(
        gpus=1,
        checkpoint_callback=checkpoint_callback,
        default_save_path=log_dir,
        train_percent_check=0.1,
    )
``` 
In my opinion that's a really bad default behaviour as the drive will blow up saving a checkpoint each epoch.

#### What's your environment?   
- python 3.7.4
- torch 1.3.0  
- Lightning 0.5.2.1
- Test-tube 0.7.1

**Comment by darleybarreto** [comment]
I did not understand your question, could you explain a bit more what you are trying to do? I might be able to help you.

**Comment by fellnerse** [comment]
How would you make lightning only save the best checkpoint per run, with as little code as possible? And not changing the folder where the checkpoint is placed (inside the logging dir).

**Comment by williamFalcon** [comment]
that’s the default behavior if you don’t pass in a checkpoint callback

**Comment by fellnerse** [comment]
Mhm maybe I understand something wrong. But if I run:
```
trainer = Trainer()
trainer.fit(model)
```
https://github.com/williamFalcon/pytorch-lightning/blob/master/pytorch_lightning/trainer/trainer.py#L282 here the default callback will instantiate ModelCheckpoint without any parameter for `save_best_only`.  And in init of ModelCheckpoint https://github.com/williamFalcon/pytorch-lightning/blob/master/pytorch_lightning/callbacks/pt_callbacks.py#L178 the default for `save_best_only` is False.
Basically my question is how can I just make it save only the best one. But don't touch the code with creating new folders for logging etc...because this works really nicely.

Btw. I really enjoy using this tool.
